litmus-portal/graphql-server/pkg/database/mongodb/operations.go

```diff
@@ -166,15 +166,14 @@ func InsertChaosWorkflow(chaosWorkflow ChaosWorkFlowInput) error {
 
 func DeleteChaosWorkflow(workflowid string) (bool, error) {
 	ctx, _ := context.WithTimeout(backgroundContext, 10*time.Second)
-	res, err := workflowCollection.DeleteOne(ctx, bson.M{"workflow_id": workflowid})
+	update := bson.M{"$set": bson.M{"isRemoved": true}}
```

Comment:
The workflow needs to be deleted from the cluster also, or else it will keep generating the new workflow runs in case of scheduled workflow

Reply:
cc: @rajdas98

Reply:
@gdsoumya We are deleting the manifest too, will that not work?

Reply:
I think for this we have to change the logic in the upsert function. Its entry is not present don't insert it.

Reply:
func UpdateWorkflowRun

Reply:
No I meant deleting from the cluster where the workflow is running, just deleting from db doesn't stop the workflow from running.

Reply:
Deleting from cluster functionality is present